CHANGELOG.md

```diff
@@ -8,7 +8,8 @@
 
 #### Android
 
-* **[Fix]** Updated Google Play Services and Firebase to v71 versions in Push module.
+* **[Breaking change]** We Updated Google Play Services and Firebase to v71 versions in Push module which forces applications to use Android 9.0 Mono frameworks.
```

Comment:
This is only in App Center Push, not in core.
There is not always a packages.config (s) file if you use project.json or modern csproj.